src/hivtrace.js

```diff
@@ -4,6 +4,7 @@ import { hivtrace_cluster_graph_summary } from "./hivtrace_cluster_graph_summary
 import { histogram, histogramDistances } from "./histogram.js";
 import { scatterPlot } from "./scatterplot.js";
 
+// QUESTION: should these be exported? (seems like they are not being used - misc, helpers, colorPicker)
```

Comment:
Yes, keep them exported for now. They may be used in other contexts. I know that colorPicker is used by @stevenweaver

Reply:
Yes, I definitely use them.

Reply:
Will do, thank you!